curator/README.md

```diff
@@ -100,7 +100,7 @@ If you want to be exact with curator, it is best to use `days` e.g. `delete: day
 [Curator issue](https://github.com/elastic/curator/issues/569)
 
 ### Compatibility with OpenShift 3.7
-In earlier releases admins could control the timezone and time when curator runs from the curator config. This configuration has been moved to openshift-ansible. The [timezone is not currently configurable](https://github.com/kubernetes/kubernetes/issues/47202), instead the timezone of OpenShift master node is used.
+In earlier releases admins could control the timezone and time when curator runs from the curator config. This configuration has been moved to openshift-ansible. The [timezone is not currently configurable](https://github.com/kubernetes/kubernetes/issues/47202), instead the timezone of the node where the pod runs is used.
```

Comment:
Is it reasonable to remove this section entirely since this is for 4.x

Reply:
In general, I would rather leave in the references to older versions, although I suppose github users by now know to select the appropriate branch/tag for the release they are using.

Reply:
I only highlight removing this section because we no longer rely on ansible and might as well remove the section which is no longer relevant in 4x.

Reply:
:+1: good way of thinking. Yes we should.